modules/client-configuration/pages/client-upgrades-sp-migration.adoc

```diff
@@ -1,23 +1,31 @@
 [[client-upgrades-spmigration]]
 = Service Pack Migration
 
-Service Pack Migration (SP Migration) allows you to upgrade SLE-based client systems from an SP level to a later one.
+Service Pack Migration (SP Migration) allows you to upgrade SLE-based client systems from an SP level to a later one, for example:
+
+* {sle}{nbsp}15{nbsp}SP1 to SP2
```

Comment:
```suggestion
Service Pack (SP) migration allows you to upgrade SLE-based client systems from an SP level to a later one.
For example, you can migrate {sle}{nbsp}15{nbsp}SP1 to {sle}{nbsp}15{nbsp}SP2.
```

Reply:
Yes, that's probably the save way.  In the past, we were told that it is possible to skip an SP.  For example, to upgrade from SP0 to SP2.  `zypper migration` and also the Web UI (SP migration) sometimes offered skipping as well.  I do not know whether that is still the case and we want to mention it.  @mcalmer , what do you think, please?